### DISCUSSION TITLE
Useful Links For Limelight and Two-Speed Transmission Drivetrain

### DISCUSSION THREAD

**Discussion by pangoliff** [frc5567/2023-Robot#182] [Team Posts]
Limelight Documentation:
http://docs.limelightvision.io/en/latest/getting_started.html#basic-programming 

Two-Speed Transmission API:
https://first.wpi.edu/FRC/roborio/beta/docs/java/edu/wpi/first/wpilibj/drive/DifferentialDrive.html

**Comment by MikasaSucasa** [comment]
I'll just leave this here... https://www.chiefdelphi.com/t/2020-limelight-settings/372677

**Comment by MikasaSucasa** [comment]
New post on CD about new limelight image file. Updates zooming and pan features. This flash should be done asap but the image file is 1gb.
